edgedb/asyncio_con.py

```diff
@@ -28,6 +28,7 @@
 
 from .datatypes import datatypes
 from .protocol import asyncio_proto
+from .protocol.protocol import IoFormat
```

Comment:
Everything looks good, one nit: can you replace `from .protocol.protocol import IoFormat` with `from .protocol import protocol` and use `IoFormat` as `protocol.IoFormat.BINARY`?

The reason is that we always import modules in all our Python projects.

Reply:
Okay. Fixed.